content/applications/finance/accounting/fiscal_localizations/localizations/italy.rst

```diff
@@ -0,0 +1,66 @@
+===============
+Italy (Sdicoop)
+===============
+
+**Cette documentation est en cours de finalisation**
```

Comment:
> Can you check what is still needed to merge this?

@jco-odoo Well, this? :p

Reply:
Our PO, @tsb-odoo would like for this documentation to be merged, even if it's not finalized yet. We are gathering informations to make sure that it is complete, but would like users to be able to consult it, but be warned that it might still change.